checks/evaluation/permissions/permissions.go

```diff
@@ -325,21 +325,21 @@ func calculateScore(result map[string]permissions) int {
 		// status: https://docs.github.com/en/rest/reference/repos#statuses.
 		// May allow an attacker to change the result of pre-submit and get a PR merged.
 		// Low risk: -0.5.
-		if permissionIsPresent(perms, "statuses") {
+		if permissionIsPresentInTopLevel(perms, "statuses") {
```

Comment:
Thanks. I can't comment on these lines, so I just picked a random line, but can we clarify the messages here. Something short mentioning least privilege working to match the checks.md file?

https://github.com/ossf/scorecard/blob/066a4f5848cfdf0680879be027f8879b0ea93e22/checks/evaluation/permissions/permissions.go#L57-L64

When I run this PR against `ossf/scorecard` I get the following, even though we have some write permissions.
```
      "score": 10,
      "reason": "tokens are read-only in GitHub workflows",
      "name": "Token-Permissions",
```

Reply:
Done, PTAL.